src/circular_buffer.jl

```diff
@@ -154,25 +154,21 @@ end
 
 Return the number of elements currently in the buffer.
 """
-Base.length(cb::CircularBuffer) = cb.length
```

Comment:
`Base.length(::AbstractArray)` is already a generic function that calls `size`. For consistency with Julia `Base`, we can define just `size` instead.